packages/wallets/src/features/cfd/components/ModalTradeWrapper/ModalTradeWrapper.tsx

```diff
@@ -7,8 +7,57 @@ import './ModalTradeWrapper.scss';
 import { WalletText } from '../../../../components/Base';
 import QRCode from 'qrcode.react';
 import useDevice from '../../../../hooks/useDevice';
+import { TMarketTypes, TPlatforms } from '../../../../types';
+import { PlatformToTitleMapper } from '../../constants';
 
-const ModalTradeWrapper: FC<PropsWithChildren<unknown>> = ({ children }) => {
+const LinksMapper: Record<
+    TPlatforms.All,
+    {
+        android: string;
+        huawei?: string;
+        ios: string;
+    }
+> = {
+    ctrader: {
+        android: 'https://play.google.com/store/apps/details?id=com.deriv.ct',
+        ios: 'https://apps.apple.com/cy/app/ctrader/id767428811',
+    },
+    derivez: {
+        android: 'https://play.google.com/store/apps/details?id=com.deriv.app&pli=1',
+        huawei: 'https://appgallery.huawei.com/#/app/C103801913',
+        ios: 'https://apps.apple.com/my/app/deriv-go/id1550561298',
+    },
+    dxtrade: {
+        android: 'https://play.google.com/store/apps/details?id=com.deriv.dx',
+        huawei: 'https://appgallery.huawei.com/app/C104633219',
+        ios: 'https://apps.apple.com/us/app/deriv-x/id1563337503',
+    },
+    mt5: {
+        android: 'https://download.mql5.com/cdn/mobile/mt5/android?server=Deriv-Demo,Deriv-Server,Deriv-Server-02',
+        huawei: 'https://appgallery.huawei.com/#/app/C102015329',
+        ios: 'https://download.mql5.com/cdn/mobile/mt5/ios?server=Deriv-Demo,Deriv-Server,Deriv-Server-02',
+    },
+};
+
+const PlatformToLinkMapper: Record<TPlatforms.All, string> = {
+    ctrader: 'https://onelink.to/hyqpv7',
+    derivez: 'https://onelink.to/bkdwkd',
+    dxtrade: 'https://onelink.to/grmtyx',
+    mt5: 'https://onelink.to/grmtyx',
+};
```

Comment:
Can add this to PlatformToTitleMapper in constants file. Dont need duplicate
Something like this ?  

export const PlatformToTitleMapper: Record<TPlatforms.All, string> = {
 ctrader: {
title: 'Deriv cTrader',
link: 'https://onelink.to/hyqpv7',
}
   //other platforms
};

Reply:
But this is only for trade links though and used only in trade modal, and it wouldn't be a mapping to title anymore but mapping to title and links

Reply:
Yeah what i mean is like this https://github.com/binary-com/deriv-app/pull/11090

Reply:
This is done here: https://github.com/binary-com/deriv-app/pull/10981/commits/b60eea46717b89df65100f6729a8a9e8ba8efcb4